### PR TITLE
feat: add json output for backup and restore cmds

### PR DESCRIPTION
**Pull request by morningvera**
[CH33701](https://app.clubhouse.io/replicated/story/33701/kots-cli-add-output-formatting-option-for-json-and-status-field-if-possible)

### CODE REVIEW

File: cmd/kots/cli/backup.go [hunk 1]
```diff
@@ -23,20 +26,37 @@ func BackupCmd() *cobra.Command {
...
+			}
+
 			options := snapshot.CreateInstanceBackupOptions{
 				Namespace: namespace,
 				Wait:      v.GetBool("wait"),
+				Silent:    output != "",
 			}
-			if err := snapshot.CreateInstanceBackup(cmd.Context(), options); err != nil {
+			backupRes, err := snapshot.CreateInstanceBackup(cmd.Context(), options)
+			if err != nil && output == "" {
```
Comment (sgalsaleh) [inline cmd/kots/cli/backup.go:40]: what about if there's an error and an output format is specified? i don't see that handled

Comment (sgalsaleh) [reply cmd/kots/cli/backup.go:40]: should be like this one :) https://github.com/replicatedhq/kots/pull/2083/files#diff-5e5da03ddf8e728ae099106e99cc9a821f479eeaaedee622042cf98baae91626R56

Comment (morningvera) [reply cmd/kots/cli/backup.go:40]: ahhh i had already fixed that but must've lost it while splitting. good catch!

Comment (morningvera) [reply cmd/kots/cli/backup.go:41]: fixed

**Review by sgalsaleh — APPROVED** [review]

**Pull request merged**